Bottom: f7a16068016fa6e17c5b7c098f917e1de0b14d60
Top:    b55a2526089f66b6c0274ea779a6cb72377eb910
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 03:31:23 +0100

Avoid clearing pins / vias from polygons... it is already done!




---

diff --git a/src/draw.c b/src/draw.c
index 9c6d8ee..d52515f 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1238,6 +1238,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
       AddPart (Pin);
       return;
     }
+#if 0
   /* Clear the area around the pin */
   if (TEST_FLAG (SQUAREFLAG, Pin))
     {
@@ -1260,6 +1261,7 @@ ClearPin (PinTypePtr Pin, int Type, int unused)
     {
       gui->fill_circle (Output.pmGC, Pin->X, Pin->Y, half);
     }
+#endif
   if ((!TEST_FLAG (PINFLAG, Pin) && !PCB->ViaOn)
       || (TEST_FLAG (PINFLAG, Pin) && !PCB->PinOn))
     return;
